test/e2e/framework/resource_usage_gatherer.go

```diff
@@ -370,6 +371,29 @@ const (
 	MasterAndDNSNodes NodesSet = 2
```

Comment:
we should rename these constants at some point too

Master->ControlPlane

Reply:
done